PULL_REQUEST_TEMPLATE

```diff
@@ -1,3 +1,9 @@
+<!-- Uncomment this block and include PR # of change against 2.13.x where appropriate
+##### ABBREVIATED REVIEW BETWEEN 2.13.X AND MASTER IS IN EFFECT
+Link to 2.13.x PR: #XXXX
```

Comment:
I think we should mention that the rest of this template can be deleted when the PR is a forward port so that the PR description just includes
1. the ABBREVIATED REVIEW banner
2. the link to the other PR
3. the list of reviewers
4. a link to the `Notes on Review Process` page

Reply:
I can get on board with that. One sec...

Reply:
Maybe:
```
<!-- Uncomment this block and include PR # of change against 2.13.x where appropriate
##### ABBREVIATED REVIEW BETWEEN 2.13.X AND MASTER IS IN EFFECT
#### Link to 2.13.x PR
#XXXX
#### Who is reviewing it? 
#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 
____
-->
```
and then follow with the normal template

Reply:
See if you like the change I made. If not, yours is certainly clear and straightforward.